docs/src/ref_impl/cross_compilation.md

```diff
@@ -22,19 +22,25 @@ To enable ``binfmt``, we recommend to set the following in your host systems ``c
       "riscv64-linux"
       "aarch64-linux"
     ];
-    
+
 In addition, it is recommended to enable KVM support with either
 
     boot.kernelModules = [ "kvm-amd" ];
 
 or
 
     boot.kernelModules = [ "kvm-intel" ];
-    
+
 depending on whether your development host is running AMD or Intel processor.
 
+## Cross Compilation for Microchip Icicle Kit (RISCV64)
```

Comment:
Please change Cross Compilation -> Cross**-**Compilation and check that the heading is formatted correctly: two newlines before and one after (https://github.com/tiiuae/ghaf/blob/main/docs/style_guide.md#markdown-syntax).

Reply:
Done.